### PR TITLE
Minor improvements to tropical geometry code. 

### PR DESCRIPTION
**Pull request by martapanizzut**
Deal with the fallout of #1194.

### CODE REVIEW

File: src/TropicalGeometry/hypersurface.jl [hunk 1]
```diff
@@ -60,7 +60,7 @@ end
 @doc Markdown.doc"""
     TropicalHypersurface(f::AbstractAlgebra.Generic.MPoly{Oscar.TropicalSemiringElem{T}})
 
-Return the tropical hypersurface defined by a tropical polynomial.
+Returns the tropical hypersurface of a tropical polynomial `f`.
```
Comment (fingolfin) [inline src/TropicalGeometry/hypersurface.jl]: We prefer imperative form, following [the Julia conventions for this](https://docs.julialang.org/en/v1/manual/documentation/)
```suggestion
Return the tropical hypersurface of a tropical polynomial `f`.
```

Comment (fingolfin) [reply src/TropicalGeometry/hypersurface.jl]: (Obviously this applies to the rest of this PR, too)

**Review by YueRen — APPROVED** [review]

File: src/TropicalGeometry/hypersurface.jl (hunk 1 above)
Comment (martapanizzut) [reply src/TropicalGeometry/hypersurface.jl]: Thanks for pointing this out.

**Comment by fingolfin** [comment]
This is still marked as a draft. If you think this is ready for merging, please use the "Ready for review" button.

**Comment by fingolfin** [comment]
What's the status of this PR?

**Comment by fingolfin** [comment]
Ahhh.... that merge was maybe not the best idea: the tests for it run last on 2nd of April. There is a good chance that in between, something changed which broke the test. In that case, we'll have to revert this merge... 

But let's hope that it's all fine :-). Next time, I recommend to first re-run the tests if they last run so long ago

**Pull request merged**